src/game/game2048.ts

```diff
@@ -42,6 +42,10 @@ export interface IGameState {
     return this.scores;
   }
 
+  public getUserActionsQueue() {
+    return this.userActionsQueue;
+  }
```

Comment:
Just a follow up from https://github.com/TheUnblockedDemo/2048/pull/10. This feels like something we'll need eventually.

Reply:
Great, let's go ahead and do this!

Reply:
What's the delay? It's been 30 seconds! GO!

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/46dc6a99-9f01-4cac-b09d-916f0ca18645?message=3f38b479-4aaa-4f99-a447-e753ffe72d87).

Reply:
Will do!

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/46dc6a99-9f01-4cac-b09d-916f0ca18645?message=871f978b-faae-46dd-8632-ddf1b9f42623).